### DISCUSSION TITLE
Run failed: pages build and deployment - main (effb7eb)

### DISCUSSION THREAD

**Discussion by Clinhar1** [community/community#68205] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Do you know how can I solve this issue? 

with:
    source: .
    destination: ./_site
    future: false
    build_revision: effb7eb601f6b347cba8ade9540a1a0d8cf136ef
    verbose: true
    token: ***
/usr/bin/docker run --name ghcrioactionsjekyllbuildpagesv108_37d159 --label 94351b --workdir /github/workspace --rm -e "INPUT_SOURCE" -e "INPUT_DESTINATION" -e "INPUT_FUTURE" -e "INPUT_BUILD_REVISION" -e "INPUT_VERBOSE" -e "INPUT_TOKEN" -e "HOME" -e "GITHUB_JOB" -e "GITHUB_REF" -e "GITHUB_SHA" -e "GITHUB_REPOSITORY" -e "GITHUB_REPOSITORY_OWNER" -e "GITHUB_REPOSITORY_OWNER_ID" -e "GITHUB_RUN_ID" -e "GITHUB_RUN_NUMBER" -e "GITHUB_RETENTION_DAYS" -e "GITHUB_RUN_ATTEMPT" -e "GITHUB_REPOSITORY_ID" -e "GITHUB_ACTOR_ID" -e "GITHUB_ACTOR" -e "GITHUB_TRIGGERING_ACTOR" -e "GITHUB_WORKFLOW" -e "GITHUB_HEAD_REF" -e "GITHUB_BASE_REF" -e "GITHUB_EVENT_NAME" -e "GITHUB_SERVER_URL" -e "GITHUB_API_URL" -e "GITHUB_GRAPHQL_URL" -e "GITHUB_REF_NAME" -e "GITHUB_REF_PROTECTED" -e "GITHUB_REF_TYPE" -e "GITHUB_WORKFLOW_REF" -e "GITHUB_WORKFLOW_SHA" -e "GITHUB_WORKSPACE" -e "GITHUB_ACTION" -e "GITHUB_EVENT_PATH" -e "GITHUB_ACTION_REPOSITORY" -e "GITHUB_ACTION_REF" -e "GITHUB_PATH" -e "GITHUB_ENV" -e "GITHUB_STEP_SUMMARY" -e "GITHUB_STATE" -e "GITHUB_OUTPUT" -e "RUNNER_OS" -e "RUNNER_ARCH" -e "RUNNER_NAME" -e "RUNNER_ENVIRONMENT" -e "RUNNER_TOOL_CACHE" -e "RUNNER_TEMP" -e "RUNNER_WORKSPACE" -e "ACTIONS_RUNTIME_URL" -e "ACTIONS_RUNTIME_TOKEN" -e "ACTIONS_CACHE_URL" -e "ACTIONS_ID_TOKEN_REQUEST_URL" -e "ACTIONS_ID_TOKEN_REQUEST_TOKEN" -e GITHUB_ACTIONS=true -e CI=true -v "/var/run/docker.sock":"/var/run/docker.sock" -v "/home/runner/work/_temp/_github_home":"/github/home" -v "/home/runner/work/_temp/_github_workflow":"/github/workflow" -v "/home/runner/work/_temp/_runner_file_commands":"/github/file_commands" -v "/home/runner/work/carolinhart.github.io/carolinhart.github.io":"/github/workspace" ghcr.io/actions/jekyll-build-pages:v1.0.8
Configuration file: none
  Logging at level: debug
      GitHub Pages: github-pages v228
      GitHub Pages: jekyll v3.9.3
             Theme: jekyll-theme-primer
      Theme source: /usr/local/bundle/gems/jekyll-theme-primer-0.6.0
         Requiring: jekyll-github-metadata
To use retry middleware with Faraday v2.0+, install `faraday-retry` gem
             ERROR: YOUR SITE COULD NOT BE BUILT:
                    ------------------------------------
                    Invalid YAML front matter in /github/workspace/src/layouts/PostLayout.astro
         Requiring: jekyll-seo-tag
         Requiring: jekyll-coffeescript
         Requiring: jekyll-commonmark-ghpages
         Requiring: jekyll-gist
         Requiring: jekyll-github-metadata
         Requiring: jekyll-paginate
         Requiring: jekyll-relative-links
         Requiring: jekyll-optional-front-matter
         Requiring: jekyll-readme-index
         Requiring: jekyll-default-layout
         Requiring: jekyll-titles-from-headings
   GitHub Metadata: Initializing...
            Source: /github/workspace/.
       Destination: /github/workspace/./_site
 Incremental build: disabled. Enable with --incremental
      Generating...

**Comment by seyLu** [comment, marked as answer]
#### Hi @Clinhar1,

You have 3 workflows for deploying to github pages:
![image](https://github.com/community/community/assets/98249191/7dd4e403-dac1-475e-85b1-1f2b0edd7f7c)

You don't need 3, just 1 that works. You are also using astro, so just follow their docs on how to deploy astro site to github pages.

I've went and [deployed a copy of your site](https://seylu.github.io/Clinhar1/) to github pages. To get the site running, I removed the unnecessary workflows and followed the [astro docs](https://docs.astro.build/en/guides/deploy/github/).

**Reply by Clinhar1** [reply]
Hello!
thank you a lot! I managed it now by myself ;) 
I don't know how - but it works. Maybe it was a local package/extension of github actions or any other changes in the settings of the repo. Also the other images are fine now, i forgot the "/" in front of the image paht. Would you be so nice and delete the copy in your repo and the copy of the site? Thank you again for the fast response and the help.

**Comment by Clinhar1** [comment]


**Reply by seyLu** [reply]
Hi, I've unpublished the site. Kindly, please close this discussion or mark your answer as the answer.
